### ISSUE TITLE
Extracting kinetic simulation data.

### ISSUE THREAD

**Issue by Rajat9975** [MicroPhen/pyFOOMB#37]
I am using the Pyfoomb package for my master thesis project. 
I am wondering how I can get the data after the kinetic simulation of the model. I do not want graphs. I need the data. Is it possible with Pyfoomb? 
Thanks in advance.

**Comment by kossikater** [comment]
Yes, when running a forward simulation using the `Caretaker` object, the return variable is a list of `TimeSeries` instances (and subclasses thereof). These have several properties, e.g. name, replicate_id, timepoint, values and thereby, each state can be identified.
The static `Helpers` class comes with a convenience method named `extract_time_series` to extract a specific `TimeSeries` object by its name and replicate_id. 
Please see the example notebook 15 how this is used by the function `investigate_last_values`, which is defined in this notebook in the top.

**Comment by kossikater** [comment]
@MicroPhen Now further responses so far, thus can be closed
